var/spack/repos/builtin/packages/molgw/package.py

```diff
@@ -18,22 +18,23 @@ class Molgw(MakefilePackage):
     MOLGW employs standard Gaussian basis set.
     """
 
-    homepage = "https://github.com/bruneval/molgw"
-    url = "https://github.com/bruneval/molgw/archive/v3.2.tar.gz"
-    git = "https://github.com/bruneval/molgw.git"
+    homepage = "https://github.com/molgw/molgw"
+    url = "https://github.com/molgw/molgw/archive/v3.3.tar.gz"
+    git = "https://github.com/molgw/molgw.git"
 
     maintainers("bruneval")
 
     license("GPL-3.0-only")
 
+    version("3.3", sha256="ff1c8eb736049e52608d4554a2d435ee9d15e47c4a9934d41712962748929e81")
     version("3.2", sha256="a3f9a99db52d95ce03bc3636b5999e6d92b503ec2f4afca33d030480c3e10242")
 
     variant("openmp", default=False, description="Build with OpenMP support")
     variant("scalapack", default=False, description="Build with ScaLAPACK support")
 
     depends_on("blas")
     depends_on("lapack")
-    depends_on("libxc@5:")
+    depends_on("libxc")
```

Comment:
Why did we remove the minimum version?

Reply:
It's a mistake. Let me revert this in the next commit